### ISSUE TITLE
Pipeline: Checking accessibility requirements should be part of our piple

### ISSUE THREAD

**Issue by GerbenBosschieter** [teamforus/general#243]
Learn more about change requests here: https://bit.ly/39CWeEE

### Requested by:
- The dutch law, team members and municipalities.  

### Change description
- We should start including checking the accessibility requirements into our pipeline.
For now we are compliant with the law because we have accessibility statement on our pages of the webshops.
From september we should be 100% compliant and a statement itself will not be enough to be compliant.

**Comment by maxvisser** [comment]
I like this CR alot :) I would like to extend the scope of this a bit:
- Training for Designers/programmers/reviewers about accessibility guidelines
- A good checklist for approving new design elements

**Comment by GerbenBosschieter** [comment]
Big scope :)

**Comment by jamalv** [comment]
Really great CR indeed; I think the right place would be during "prototyping" in figma, so that's after the IP, and before the epic.

**Comment by jamalv** [comment]
And let's start small.. I added a accessibility review icon for the prototyping phase.. We discussed that gerben would take this task on himself.. We can start with just checking the basics, and then work on extending the scope from there.. Having this in the pipeline will already start to raise the awareness level

<img width="380" alt="Screenshot 2020-06-19 at 20 47 34" src="https://user-images.githubusercontent.com/30194799/85170742-1d8abe00-b26e-11ea-8059-fcf2c57324ec.png">

**Comment by GerbenBosschieter** [comment]
Some of accessibility guidelines are also related to html. @maxvisser maybe we can think of something that helps us to include these guidelines in our code review process. 

When doing different quickscans for accessibility you will get the results that references to the related guideline.
For start we could include these guidelines everytime we review code. This way it's easier maintainable that reviewing everything at once and it will be a start to include this in our process.

**Example guidelines**
- image references should have alternative text
- html structure should be readable for text to speech

easy stuff

**Comment by maxvisser** [comment]
Yes I agree. I would still like to focus first on aswell making a guide for this to reference to; example of a guide:
https://github.com/fejes713/accessibility-guide 

So each new page/feature can be checked based upon a set of guidelines.

**Comment by maxvisser** [comment]
@SjoerdHilhorst Maybe you find this interesting aswell. I looked into Lighthouse (a chrome extension) which does a accessibility audit and we score pretty poor. Some things are pretty easy to fix (like gerben described, a alt text for an image which is missing). 

Used this tool:
https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=nl

Results:
https://drive.google.com/file/d/1DjqDPrXroX9nfdXhS35bSI5wbAWqk1DA/view?usp=sharing

**Comment by maxvisser** [comment]
There is aswell an angular plugin called ngAria (angular Accessible Rich Internet Application) which is built by google to support screen readers. If we have the basics done, we could implement ARIA on the requester signupflow and voucher page

**Comment by jamalv** [comment]
For now, Gerben will do a review @GerbenBosschieter could you list here what you included in your review? This way we can use it to create a manual for the designers and devs.

**Comment by jamalv** [comment]
Yesterday we had a presentation about this. There are a few tasks to this epic I see, that I think can be planned:
- [ ] Don't accept designs without an accessibility check (create/update internal policy)
- [ ] Make WCAG requirements part of dev requirements: Contrast, HTML structure, Alternative tekst
- [ ] Make a list of essential plugins to check accesibility and make it required for devs to use these plugins
- [ ] Research options of including automated testing into the pipeline
- [ ] Update testing to test on accessibility - (for new functionality test the functionality with the release, and a full test every quarter?)

**Comment by maxvisser** [comment]
lets move this CR to ITSM
